azkaban-common/src/main/java/azkaban/executor/IntHandler.java

```diff
@@ -0,0 +1,24 @@
+package azkaban.executor;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.apache.commons.dbutils.ResultSetHandler;
+
+public class IntHandler implements ResultSetHandler<Integer> {
+  public static final String NUM_EXECUTIONS =
```

Comment:
The string variables should probably be defined in respective dao class as private static field and not in this generic class.

Reply:
I am removing this file.. This file change is not needed.